web/src/components/MobileBanner.tsx

```diff
@@ -51,14 +50,16 @@ const StyledDivText = styled.div`
 
 const StyledAppName = styled.span`
   font-weight: bold;
-  font-size: 14px;
+  font-size: 12px;
   color: ${props => props.theme.colors.themeContrast};
 `
 
-const StyledDescription = styled.span`
+const smallScreenSize = 400
```

Comment:
Please use either our hook

 ``` tsx
const { viewportSmall } = useWindowDimensions()
```
or a media query:
``` css
@media ${dimensions.smallViewport} {
...
}
```

Reply:
![image](https://github.com/user-attachments/assets/0a2b1cbf-d565-46d5-a6a5-775ce55c0aa7)
smallViewport is 768px.. I need smaller size for iphone SE for example

Reply:
Also the banner shows up at the smallViewport so there is no need for media query at this point.

Reply:
Sorry, I  missed that. Thanks for explaining.